src/photos/src/NIPhotoScrollView.m

```diff
@@ -103,10 +57,15 @@ - (id)initWithFrame:(CGRect)frame {
     self.doubleTapToZoomIsEnabled = YES;
 
     // Autorelease so that we don't have to worry about releasing the subviews in dealloc.
-    _scrollView = [[NICenteringScrollView alloc] initWithFrame:self.bounds];
+    _scrollView = [[UIScrollView alloc] initWithFrame:self.bounds];
     _scrollView.autoresizingMask = (UIViewAutoresizingFlexibleWidth
                                     | UIViewAutoresizingFlexibleHeight);
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
```

Comment:
This should be:
```
#if defined(__IPHONE_11_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0
```

Reply:
I think this comment still needs to be addressed?

Reply:
I removed changes to contentInsetAdjustmentBehavior from this PR, so this comment no longer has to be addressed.